src/System.Private.ServiceModel/tools/SelfHostedWcfService/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Web.Administration" version="7.0.0.0" targetFramework="net451" userInstalled="true" />
```

Comment:
Both Self-host and IIS host have CRL restful service. The difference is IIS host require some extra dependencies.